scripts/release-binary.sh

```diff
@@ -40,18 +40,25 @@ DST=""
 # Verify that we're building binaries on Ubuntu 16.04 (Xenial).
 CHECK=1
 
+# Push envoy docker image.
+PUSH_DOCKER_IMAGE=1
+
 function usage() {
   echo "$0
     -d  The bucket name to store proxy binary (optional).
+        If not provided, both envoy binary push and docker image push are skipped.
     -i  Skip Ubuntu Xenial check. DO NOT USE THIS FOR RELEASED BINARIES.
-        Cannot be used together with -d option."
+        Cannot be used together with -d option.
+    -s  Skip pushing envoy docker image. This should only be useful if -d is provided,
```

Comment:
Since pushing to GCS is optional, this should be as well, for consistency.

Could you change this to `-p` (push) and default to false?